packages/esm-patient-chart-app/src/visit/visits-widget/visit-detail-overview.scss

```diff
@@ -47,6 +47,18 @@
   }
 }
 
+.header {
+  .visitInfo {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    .visitActions {
+      border-left: 1px solid $color-gray-30; 
+      padding-left: 3rem;
```

Comment:
Last couple of things:

1. In general, it's preferable to use Carbon's `spacing` variables (i.e., `spacing.$spacing-09` matches here).
2. However, in similar components (other tables) where we have this we use this:

SCSS:

```scss
.divider {
  width: 1px;
  height: spacing.$spacing-05;
  color: $ui-03;
  margin: 0rem spacing.$spacing-03 0 spacing.$spacing-05;
}
```

JSX:

```jsx
  <span className={styles.divider}>|</span>
  <Button
    kind="ghost"
    renderIcon={(props) => <Add size={16} {...props} />}
    iconDescription="Add biometrics"
    onClick={launchBiometricsForm}
  >
    {t('add', 'Add')}
  </Button>
```

For visual consistency, we should use this rather than a border line.